script/eval.go

```diff
@@ -60,16 +60,39 @@ func evalSnapshot(r *runner, n node) error {
 func evalUSE(r *runner, n node) error {
 
 	imageName := n.args[0]
-	replaceImgs := make([]string, len(n.args)-1)
-	//if len(n.args) > 1 && n.args[1] != "" {
+	replaceImgs := make([]string, 0)
```

Comment:
I know this code is poorly commented, but add a comment to the effect of:

If "service" is in the SVC_USE command the image will be replaced only on the service (and its children) listed after the "service" keyword.  eg.  `SVC_USE [PREVIOUS_IMAGE_NAMES] service <SVC_PATH>`

Reply:
Please add comments on the expected format of the updated command